### DISCUSSION TITLE
Welcome to 100DaysOfHacking Discussions!

### DISCUSSION THREAD

**Discussion by 0xk4b1r** [0xk4b1r/cybersecurity#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by looloolove** [comment]
Hi! 😀 My name is Laura and I'm SUPER EXCITED to be here! I am here to learn everything that I possibly can about the project.  I'm a total newbie...so please be patient with me!! 🤪

**Reply by 0xk4b1r** [reply]
Hi Laura, Glad to see you here.

**Reply by looloolove** [reply]
Thank you! I'm so happy to be here!
